src/main/java/Service/AccountService.java

```diff
@@ -0,0 +1,53 @@
+package Service;
+
+import Entity.User;
+import DAO.UserDAO;
+
+public class AccountService {
+
+    private User user;
+    private boolean isLogin;
```

Comment:
maybe better add this field at model ant set values true or false here

Reply:
Does you tell about Controller ?